### ISSUE TITLE
[FEATURE] Vertical split tabs

### ISSUE THREAD

**Issue by agilecohix** [zeit/hyper#1908]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  Windows 10 Creator's
- **Hyper.app version**: 1.3.3
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: N/A
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: N/A
## Issue

While the split pane functionality is good, I think it would be quite useful to have the ability to put tabs side by side. Eventually it would be nice to have draggable tabs, tearable tabs, and tabs that snap to top/bottom/sides of the hyper window, similar to Atom/VS Code/Sublime/etc.

Thanks for the great product, cheers!

**Comment by CodeTheory** [comment]
For the record, you're talking about panes with their own tab set? Like Atom?
I like this idea!

**Comment by agilecohix** [comment]
@CodeTheory Yes exactly! That would be extremely useful for many different workflows.

**Comment by Anzumana** [comment]
i love the feature too but not sure if this would be the best way to implement this as tmux has these features already.
i myself run tmux inside of hyper to get all the panes my heart could desire :).
what i was thinking about was starting a tmux integration of hyper something similar to what the iterm team did but for hyper. 
Thoughts ?
